apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -67,6 +68,25 @@ function strip(code) {
   );
 }
 
+/**
+ * Given a type string for a field input, returns an appropriate change handler function
+ * for that type, which customizes the input field and provides validation on blur.
+ * @param {string} type
+ * @returns {?function}
+ */
+function getFieldInputChangeHandler(type) {
+  const clampedNumberMatch = type.match(CLAMPED_NUMBER_REGEX);
+  if (clampedNumberMatch) {
+    const min = parseFloat(clampedNumberMatch[1]);
+    const max = parseFloat(clampedNumberMatch[2]);
+    return Blockly.FieldTextInput.clampedNumberValidator(min, max);
```

Comment:
@mikeharv is there any pattern to when we use global capital B Blockly in the wrapper? It seems a little weird to me that it gets used in the wrapper at all, since we're constructing an object that is eventually going to be assigned to window.Blockly. Wouldn't it be more appropriate to use `blocklyInstance` (or `this`, maybe)? Probably doesn't really matter, just curious.

Reply:
I don't think we have a convention established here, unfortunately. Does `this` work here? If so, that seems like it might be fairly readable.